On The Map/Controller/MapKitViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  MapKitViewController.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/25/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class MapKitViewController: UIViewController, MKMapViewDelegate {
+    
+    var students:[Student] = []
+
+    @IBOutlet weak var mapView: MKMapView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mapView.delegate = self
+        self.pinLocatinDisplay()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
```

Comment:
you are passing `true` to super here regardless of whether `true` or `false` was passed to you. You should just pass the value of `animated` to the super call.